console/src/main.rs

```diff
@@ -119,8 +126,12 @@ async fn main() -> color_eyre::Result<()> {
                     .push(Span::styled(" PAUSED", view.styles.fg(Color::Red)));
             }
             let header = Paragraph::new(header_text).wrap(Wrap { trim: true });
+            let view_controls =
+                Paragraph::new("views: Tasks (F1), Resources (F2)").wrap(Wrap { trim: true });
```

Comment:
nit: would be nice if this was formatted more like the other controls text (with the keys in bold, delimited by equals signs), for consistency.